src/components/modal/modal.scss

```
@@ -61,7 +61,7 @@ $zIndex: 11;
     &-close {
         position: absolute;
         top: 2%;
-        right: 2%;
+        right: 10%;
```

10% это много, давай сделаем 9?
```suggestion
        right: 9%;
```